corehq/apps/api/resources/v0_5.py

```diff
@@ -264,7 +269,7 @@ def _update(self, bundle, user_change_logger=None):
                             groups = [Group.wrap(doc) for doc in get_docs(Group.get_db(), group_ids)]
                         user_change_logger.add_info(UserChangeMessage.groups_info(groups))
                     should_save = True
-                elif key in ['email', 'username']:
+                elif key == 'email':
```

Comment:
Nice!
good to see that work for user change logger helped point these things out, https://dimagi-dev.atlassian.net/browse/SUPPORT-10484.